test/util/memwallet.js

```diff
@@ -56,11 +56,6 @@ class MemWallet {
       this.key = options.key;
     }
 
-    if (options.witness != null) {
-      assert(typeof options.witness === 'boolean');
-      this.witness = options.witness;
-    }
-
```

Comment:
`deriveKey` later here passes `witness` option to `Keyring` which we removed earlier. Should be fixed.

Reply:
yep, option will not be parsed by keyring. and memwallet witness will always be false